lib/quandl/request.rb

```diff
@@ -1,7 +1,7 @@
 require 'open-uri'
 
 module Quandl
-  API_URI = 'http://www.quandl.com/api/'
+  API_URI = 'https://www.quandl.com/api/'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.